sql/select.go

```diff
@@ -468,13 +256,13 @@ func (v *indexInfo) init(s *scanNode) {
 
 // analyzeRanges examines the range map to determine the cost of using the
 // index.
-func (v *indexInfo) analyzeRanges(m qvalueRangeMap) {
+func (v *indexInfo) analyzeRanges(exprs []parser.Exprs) {
 	if !v.covering {
 		return
 	}
 
-	v.makeStartInfo(m)
-	v.makeEndInfo(m)
+	v.makeStartInfo(exprs)
```

Comment:
Probably, though I don't want to change this code much in this PR. See the TODO added in one of the commits to `planner.selectIndex` about where this code is going.